internal/sdkv2provider/schema_cloudflare_zone_cache_reserve.go

```diff
@@ -0,0 +1,29 @@
+package sdkv2provider
+
+import (
+	"github.com/cloudflare/terraform-provider-cloudflare/internal/consts"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceCloudflareZoneCacheReserveSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		consts.ZoneIDSchemaKey: {
+			Type:        schema.TypeString,
+			Required:    true,
+			ForceNew:    true,
+			Description: consts.ZoneIDSchemaDescription,
+			ValidateFunc: func(value any, key string) (_ []string, errs []error) {
+				// Ensure that a valid Zone ID was passed.
+				if err := validateZoneID(value.(string)); err != nil {
+					errs = append(errs, err)
+				}
+				return
+			},
+		},
+		"enabled": {
```

Comment:
**To the reviewer:**

I made this to be a **required** attribute on purpose, as this has both a monetary value attached to it and performance side-effects when enabled (or disabled), especially since it requires a dedicated pricing plan to be enabled, and as such user has to explicitly choose to enable or disable this feature of their zone.

Thus, no default setting is being offered here.